packager.go

```diff
@@ -43,6 +43,16 @@ const userAgent = "Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) " +
 	"AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.96 Mobile " +
 	"Safari/537.36 (compatible; amppackager/0.0.0; +https://github.com/ampproject/amppackager)"
 
+// Conditional request headers that ServeHTTP may receive and need to be sent with fetchURL.
```

Comment:
This change seems significant enough to warrant a unittest. If you're ambitious, try adding one to packager_test.go. This will require changing the mock CDN that's set up in TestMain. (TestMain is a setup function that gets called once before all tests are run. Go doesn't have a traditional per-test "setup" method -- just include that code in the test itself.) If you're not ambitious, file a bug to implement the test. :)

Reply:
git is acting up again so I want to be done with this branch. I've filed a bug #45 to address this feedback.